src/main/java/reactor/netty/http/server/ConnectionInfo.java

```diff
@@ -152,6 +158,16 @@ static ConnectionInfo parseXForwardedInfo(HttpRequest request, SocketChannel cha
 		return new ConnectionInfo(hostAddress, remoteAddress, scheme);
 	}
 
+	static InetSocketAddress getRemoteAddress(SocketChannel channel) {
+		HAProxyMessage proxyMessage = channel.attr(PROXY_PROTOCOL_MESSAGE).get();
```

Comment:
Once obtaining the information, do we need this attribute or we can remove it?

Reply:
Yes, it can be removed.